### PR TITLE
Update Access Request instructions for Redmine Issue Template use

### PR DESCRIPTION
**Pull request by djgalloway**
Signed-off-by: David Galloway <dgallowa@redhat.com>

### CODE REVIEW

File: doc/adding_users.rst [hunk 1]
```diff
@@ -1,23 +1,35 @@
...
-    ssh-keygen -t rsa
-
-#. A hashed VPN password.  See below for how to set this up:
+#. `File a ticket <http://tracker.ceph.com/projects/lab/issues/new>`_.
+
+#. Set **Tracker** to "Support"
+
+#. Set **Issue template** to "Sepia Lab Access Request."
+
+   **NOTE:** you may need to disable javacript-blocking browser plugins.
```
Comment (dmick) [inline doc/adding_users.rst]: javacript

**Comment by dmick** [comment]
otherwise :+1:

File: doc/adding_users.rst (hunk 1 above)
Comment (djgalloway) [reply doc/adding_users.rst]: Whoops.  Good catch.  Fixed!

**Comment by djgalloway** [comment]
@dmick ping

**Pull request merged**